File: annotation_precessing/src/main/java/com/hy/annotation_precessing/MainActivity.java
package com.hy.annotation_precessing;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

import com.hy.annotation.BindView;
import com.hy.annotation_lib.Binding;


public class MainActivity extends AppCompatActivity {
    @BindView(R.id.text)
    TextView textView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Binding.bind(this);

        textView.setText("成功");
    }
}

File: hiltsimple/src/main/java/com/hy/hiltsimple/MainActivity.kt
package com.hy.hiltsimple

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import dagger.hilt.android.AndroidEntryPoint
import javax.inject.Inject


@AndroidEntryPoint
class MainActivity : AppCompatActivity() {
    lateinit var textView: TextView

    @Inject lateinit var user: User

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        textView = findViewById(R.id.textView)

//        textView.text = "${user.name}初次使用Hilt的心情是：${user.mood}"

    }
}
